R/derive_var_ontrtfl.R

```diff
@@ -136,7 +141,24 @@
 #'  ref_end_window = 60,
 #'  span_period = "Y"
 #' )
+#'
+#' advs <- tibble::tribble(
+#'   ~USUBJID, ~ASTDT,              ~AP01SDT,           ~AP01EDT,           ~AENDT,
+#'   "P01",    ymd("2020-03-15"), ymd("2020-01-01"), ymd("2020-03-01"), ymd("2020-12-01"),
+#'   "P02",    ymd("2019-04-30"), ymd("2020-01-01"), ymd("2020-03-01"), ymd("2020-03-15"),
+#'   "P03",    ymd("2019-04-30"), ymd("2020-01-01"), ymd("2020-03-01"), NA,
+#' )
+#' derive_var_ontrtfl(
+#'  advs,
+#'  new_var = "ONTR01FL",
+#'  start_date = ASTDT,
+#'  end_date = AENDT,
+#'  ref_start_date = AP01SDT,
+#'  ref_end_date = AP01EDT,
+#'  span_period = "Y"
+#' )
 derive_var_ontrtfl <- function(dataset,
+                               new_var = "ONTRTFL",
```

Comment:
This should be a symbol, i.e. `ONTRTFL`, not a string, i.e. `"ONTRTFL"`. Please take a look at `derive_var_base()` for an example of how this works (it's a bit more tricky than using strings.